### DISCUSSION TITLE
Table Header Css

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#9481] [CLI - PWA mode] [answered]
[table header.txt](https://github.com/quasarframework/quasar/files/6566689/table.header.txt)
Hello friends 
 Am having some issue to implement Css on q-table . i want my table header to display like this 
![image](https://user-images.githubusercontent.com/84211129/120115267-f6ce4a80-c193-11eb-8b48-e6573d32928a.png)

Can some please help thanks

**Comment by ontwikkelfabriek** [comment, marked as answer]
@akashramtohul4 

Here's a codepen:
https://codepen.io/ontwikkelfabriek/pen/KKWyQLB

**Comment by akashramtohul4** [comment]
Hi dobbel

Thanks its working 😁
